src/test/lambda/explorer/cloudFormationNodes.test.ts

```diff
@@ -287,7 +286,7 @@ describe('DefaultCloudFormationNode', () => {
     })
 
     it('handles error', async () => {
-        class ThrowErrorDefaultCloudFormationNode extends DefaultCloudFormationNode {
+        class ThrowErrorCloudFormationNode extends CloudFormationNode {
```

Comment:
Is this necessary? This doesn't test anything CloudFormationNode specific--this would do better a a test for AWSTreeErrorHandlerNode

Reply:
There's room for improvement in that area of testing, across all nodes. It is beyond the scope of this change.